backend/src/shared/interfaces/response-objects/BoardResponseObject.ts

```diff
@@ -1,4 +1,4 @@
-export interface BoardResponseObject {
+export class BoardResponseObject {
```

Comment:
I think there is no need to use class here. Classes are useful in combination with transformer and validator package to determine the correct dto is passed. Simple interface should be good enough. 

https://www.reddit.com/r/reactnative/comments/me9vn0/typescript_use_a_class_or_an_interface_for_api/

https://stackoverflow.com/questions/53531488/why-do-we-need-dtos-and-interfaces-both-in-nestjs

Reply:
When `BoardResponseObject` is an interface:

![image](https://github.com/user-attachments/assets/f9e28008-ab4e-4a34-a14c-350a0a3a7545)

Reply:
When leaving class instead of interface it does not work properly (i think all the fields are optional then)

Reply:
So we need both class and interface I guess

Reply:
As two separate object. What do you think? I don't really like it but can't see other way

Reply:
Please check this @marcepanowyy : https://github.com/OBELISK-TEAM/OBELISK/compare/90a081ff7922e707b00d646ba3917e5b04a66ad1..4a94e22d6609fb4fdec5779a3e2a754ee7327638

Reply:
hmm i dont like this approach

Reply:
you could try adding @ApiProperty in response objects
```
  @ApiProperty({
    description: 'descr',
    type: [sth],
  })
  prop: sth[];

```

Reply:
Yeah, thats the plan for later, but I still doesn't solve the problem.

Reply:
I guess we can stick to the class instead of interface... I see no other solution for that rn :((